src/test/java/CalculatorTest.java

```diff
@@ -0,0 +1,108 @@
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.NullAndEmptySource;
+import org.junit.jupiter.params.provider.ValueSource;
+import static org.assertj.core.api.Assertions.*;
+import static org.junit.jupiter.api.Assertions.*;
+import static org.junit.platform.commons.util.StringUtils.isBlank;
+
+@DisplayName("문자열 사칙 연산 계산기 구현")
+public class CalculatorTest {
+
+    @Test
+    @DisplayName("덧셈 테스트")
+    public void additionTest() {
+        assertThat(addition(10, 5)).isEqualTo(15);
+    }
+
+    private int addition(int a, int b){
+        return  a+b;
+    }
+
+    @Test
+    @DisplayName("뺄셈 테스트")
+    public void subtractionTest() {
+        assertThat(subtraction(10, 5)).isEqualTo(5);
+    }
+
+    private int subtraction(int a, int b){
+        return a - b;
+    }
+
+    @Test
+    @DisplayName("곱셈 테스트")
+    public void multiplicationTest() {
+        assertThat(multiplication(10, 5)).isEqualTo(50);
+    }
+
+    private int multiplication(int a, int b){
+        return a * b;
+    }
+
+    @Test
+    @DisplayName("나눗셈 테스트")
+    public void divisionTest(){
+        assertThat(division(10,5)).isEqualTo(2);
+    }
+
+    private int division(int a, int b){
+        return a / b;
+    }
+
+    @ParameterizedTest
+    @NullAndEmptySource
+    @ValueSource(strings = {"", "  "})
+    @DisplayName("입력값 NULL 또는 빈 공백 문자일 경우 예외 테스트")
+    public void chkIsNullOrBlankTest(String input) {
+        assertThatIllegalArgumentException().isThrownBy(()->{
+            int val = calculator(input);
+        }).withMessageMatching("input data is blank or null");
```

Comment:
저도 처음에는 예외처리 테스트를 코딩하신 것처럼 했었는데요,
아래 처럼 하는게 훨씬 가독성도 좋고 일반적이라고 하더라구요.
이건 개인적인 생각입니다.😁
```suggestion
        assertThatThrownBy(() -> calculator(input))
                .isInstanceOf(IllegalArgumentException.class);
```